.gitlab-ci.yml

```diff
@@ -1,5 +1,4 @@
 # yamllint disable rule:line-length
-# SPDX-License-Identifier: AGPL-3.0+
```

Comment:
Officially a no-no, but since this was added by mistake (a blind copy-paste action) and is of no use outside of UM's closed build environment, I think we can correct this license usage.